### DISCUSSION TITLE
Site Update SSL installation causes PHP fatal error

### DISCUSSION THREAD

**Discussion by afruit** [EasyEngine/easyengine#1724] [Support Requests]
When I try and install an SSLcert on an existing site, I get this:

# ee site update example.com --ssl=le
Starting SSL update for: example.com
PHP Fatal error:  Uncaught Error: Undefined constant "GuzzleHttp\IDNA_DEFAULT" in phar:///usr/local/bin/ee/vendor/guzzlehttp/guzzle/src/Client.php:218
Stack trace:
#0 phar:///usr/local/bin/ee/vendor/guzzlehttp/guzzle/src/Client.php(112): GuzzleHttp\Client->buildUri()
#1 phar:///usr/local/bin/ee/vendor/guzzlehttp/guzzle/src/Client.php(129): GuzzleHttp\Client->sendAsync()
#2 phar:///usr/local/bin/ee/vendor/acmephp/core/Http/SecureHttpClient.php(337): GuzzleHttp\Client->send()
#3 phar:///usr/local/bin/ee/vendor/acmephp/core/Http/SecureHttpClient.php(346): AcmePhp\Core\Http\SecureHttpClient->AcmePhp\Core\Http\{closure}()
#4 phar:///usr/local/bin/ee/vendor/acmephp/core/AcmeClient.php(312): AcmePhp\Core\Http\SecureHttpClient->request()
#5 phar:///usr/local/bin/ee/vendor/acmephp/core/AcmeClient.php(89): AcmePhp\Core\AcmeClient->getResourceUrl()
#6 phar:///usr/local/bin/ee/vendor/acmephp/core/AcmeClient.php(334): AcmePhp\Core\AcmeClient->getHttpClient()
#7 phar:///usr/local/bin/ee/vendor/acmephp/core/AcmeClient.php(112): AcmePhp\Core\AcmeClient->requestResource()
#8 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/helper/Site_Letsencrypt.php(179): AcmePhp\Core\AcmeClient->registerAccount()
#9 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/helper/class-ee-site.php(1504): EE\Site\Type\Site_Letsencrypt->register()
#10 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/helper/class-ee-site.php(1467): EE\Site\Type\EE_Site_Command->init_le()
#11 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/helper/class-ee-site.php(1406): EE\Site\Type\EE_Site_Command->init_ssl()
#12 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/helper/class-ee-site.php(899): EE\Site\Type\EE_Site_Command->www_ssl_wrapper()
#13 phar:///usr/local/bin/ee/vendor/easyengine/site-type-wp/src/WordPress.php(1425): EE\Site\Type\EE_Site_Command->update_ssl()
#14 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/helper/class-ee-site.php(456): EE\Site\Type\WordPress->update_ssl()
#15 [internal function]: EE\Site\Type\EE_Site_Command->update()
#16 phar:///usr/local/bin/ee/php/EE/Dispatcher/CommandFactory.php(96): call_user_func()
#17 [internal function]: EE\Dispatcher\CommandFactory::EE\Dispatcher\{closure}()
#18 phar:///usr/local/bin/ee/php/EE/Dispatcher/Subcommand.php(272): call_user_func()
#19 phar:///usr/local/bin/ee/php/EE/Runner.php(335): EE\Dispatcher\Subcommand->invoke()
#20 phar:///usr/local/bin/ee/php/class-ee.php(1184): EE\Runner->run_command()
#21 phar:///usr/local/bin/ee/vendor/easyengine/site-command/src/Site_Command.php(154): EE::run_command()
#22 [internal function]: Site_Command->__invoke()
#23 phar:///usr/local/bin/ee/php/EE/Dispatcher/CommandFactory.php(96): call_user_func()
#24 [internal function]: EE\Dispatcher\CommandFactory::EE\Dispatcher\{closure}()
#25 phar:///usr/local/bin/ee/php/EE/Dispatcher/Subcommand.php(272): call_user_func()
#26 phar:///usr/local/bin/ee/php/EE/Runner.php(335): EE\Dispatcher\Subcommand->invoke()
#27 phar:///usr/local/bin/ee/php/EE/Runner.php(358): EE\Runner->run_command()
#28 phar:///usr/local/bin/ee/php/EE/Runner.php(784): EE\Runner->_run_command_and_exit()
#29 phar:///usr/local/bin/ee/php/EE/Bootstrap/LaunchRunner.php(23): EE\Runner->start()
#30 phar:///usr/local/bin/ee/php/bootstrap.php(74): EE\Bootstrap\LaunchRunner->process()
#31 phar:///usr/local/bin/ee/php/init-ee.php(27): EE\bootstrap()
#32 phar:///usr/local/bin/ee/php/boot-phar.php(8): include('...')
#33 /usr/local/bin/ee(5): include('...')
#34 {main}
  thrown in phar:///usr/local/bin/ee/vendor/guzzlehttp/guzzle/src/Client.php on line 218
Warning: An Error occurred. Initiating clean-up.
Warning: Exiting gracefully after rolling back. This may take some time.
Success: Rollback complete. Exiting now.

Any Help is greatly appreciated.

**Comment by Cloud-Kid** [comment]
Same problem, here on a dedicated server with only EE installed, is a rollback to the previous version possible ?

**Comment by mrrobot47** [comment]
@Cloud-Kid Please update to latest version. This has already been fixed. Reference: https://github.com/EasyEngine/easyengine/issues/1722#issuecomment-1215610028

**Reply by Cloud-Kid** [reply]
Thank you so munch it was driving me insane ahah Thanks a lot for your quick answer :)
